xml/System/DateTime.xml

```diff
@@ -46,351 +46,353 @@
     <summary>Represents an instant in time, typically expressed as a date and time of day.</summary>
     <remarks>
       <format type="text/markdown"><![CDATA[  
+## Examples
+
+This section contains examples for many common uses of the `DateTime` struct:
+  
+ [Initializing a `DateTime` object](#initializing-a-datetime-object)   
+ [DateTime values and their string representations](#datetime-values-and-their-string-representations)   
+ [Parsing DateTime values from strings](#parsing-datetime-values-from-strings)   
+ [DateTime resolution](#datetime-resolution)   
+ [DateTime values and calendars](#datetime-values-and-calendars)   
+ [Persisting DateTime values](#persisting-datetime-values)   
+ [Comparing for equality within tolerance](#comparing-for-equality-within-tolerance)
+
+You can download the entire set of examples from this article in either [Visual Basic](https://github.com/docs/samples/snippets/visualbasic/System.DateTime/) or [C#](https://github.com/docs/samples/snippets/csharp/System.DateTime/) from the docs repository on GitHub.
```

Comment:
We are investigating DownGit now.  What do you think of the experience? There are other alternatives we are looking at as well. /cc @dend

Reply:
Not sure if it's DownGit's fault, but when I download the zip using DownGit, my Firefox tells me that "This file is not commonly downloaded" and that it may not be safe to open:

![](https://user-images.githubusercontent.com/287848/33572436-38439896-d933-11e7-961e-0bd2175e894b.png)

If there is another service where this doesn't happen, I would prefer that.